lib/kino_db/connection_cell.ex

```diff
@@ -244,10 +244,18 @@ defmodule KinoDB.ConnectionCell do
   end
 
   defp to_quoted(%{"type" => "sqlserver"} = attrs) do
-    quote do
-      opts = unquote(shared_options(attrs)) ++ unquote(sqlserver_options(attrs))
+    if sqlserver_options?(attrs) do
+      quote do
+        opts = unquote(shared_options(attrs)) ++ unquote(sqlserver_options(attrs))
+
+        {:ok, unquote(quoted_var(attrs["variable"]))} = Kino.start_child({Tds, opts})
+      end
```

Comment:
It would be better to do the concatenation at generation time, so we don't emit noise into the code. It will also allow us to get rid of the conditional, something like this:

```suggestion
    quote do
      opts = unquote(shared_options(attrs) ++ sqlserver_options(attrs))

      {:ok, unquote(quoted_var(attrs["variable"]))} = Kino.start_child({Tds, opts})
    end
```